app/scripts/modules/kubernetes/src/v2/pipelines/stages/runJob/KubernetesV2RunJobStageConfig.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+
+import {
+  IStageConfigProps,
+  AccountService,
+  //   HelpField,
+  YamlEditor,
+  yamlDocumentsToString,
+  IAccount,
+} from '@spinnaker/core';
+
+import { ManifestBasicSettings } from 'kubernetes/v2/manifest/wizard/BasicSettings';
+
+export interface IKubernetesRunJobStageConfigState {
+  credentials: IAccount[];
+  account: string;
+  rawManifest?: string;
+}
+
+export class KubernetesV2RunJobStageConfig extends React.Component<IStageConfigProps> {
+  public state: IKubernetesRunJobStageConfigState = {
+    credentials: [],
+    account: '',
```

Comment:
Do we need to store the selected account on state or can we simplify by reading it off the stage?